XCUITests/IntegrationTests.swift

```diff
@@ -44,4 +44,31 @@ class IntegrationTests: BaseTestCase {
         navigator.goto(SettingsScreen)
         waitforExistence(app.tables.staticTexts["Sync Now"], timeout: 10)
     }
+
+    func testFxASyncTabs () {
+        navigator.openURL("example.com")
+        waitUntilPageLoad()
+        navigator.goto(BrowserTabMenu)
+        navigator.goto(FxASigninScreen)
+        waitforExistence(app.webViews.staticTexts["Sign in"], timeout: 10)
+        userState.fxaUsername = ProcessInfo.processInfo.environment["FXA_EMAIL"]!
+        userState.fxaPassword = ProcessInfo.processInfo.environment["FXA_PASSWORD"]!
+        navigator.performAction(Action.FxATypeEmail)
+        navigator.performAction(Action.FxATypePassword)
+        navigator.performAction(Action.FxATapOnSignInButton)
+        allowNotifications()
```

Comment:
We could put lines 52-59 into a `signIntoFirefoxAccounts` method, to reduce duplication from the existing test.

Reply:
Thanks! done in PR https://github.com/mozilla-mobile/firefox-ios/pull/3967, will rebase and update this test before landing